Networking Sprint Challenge - PokeDex/Networking Sprint Challenge - PokeDex/View Controllers/DetailViewController.swift

```diff
@@ -0,0 +1,66 @@
+//
+//  DetailViewController.swift
+//  Networking Sprint Challenge - PokeDex
+//
+//  Created by Vijay Das on 2/1/19.
+//  Copyright © 2019 Vijay Das. All rights reserved.
+//
+
+import UIKit
+
+class DetailViewController: UIViewController {
+    
+    
+    var pokemonController: PokemonController?
+
+    var pokemon: Pokemon? {
+        didSet {
+            updateViews()
+        }
+    }
+
+    // update views with property observer pattern
+    
+
+    func updateViews() {
+        guard isViewLoaded else { return }
+        guard let pokemon = pokemon else { return }
+        pokeName.text = pokemon.name
+        pokeID.text = "\(pokemon.id)"
+        
+        let abilityName = pokemon.abilities.map { $0.subAbility.name}
+        let typeName = pokemon.type.map { $0.subType}
```

Comment:
You get an error here because you're calling "subType" which is not a property you created in your model. Check your model and you will see it is named something else.